File: GTTS/INL.SearchService/Models/SearchParticipants_Result.cs
using System.Collections.Generic;
using INL.Services.Models;

namespace INL.SearchService.Models
{
    public class SearchParticipants_Result : BaseResult, ISearchParticipants_Result
    {
        public List<ISearchParticipants_Item> Collection { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/search-participants_result.ts


import { ISearchParticipants_Item } from './isearch-participants_item';
import { ISearchParticipants_Result } from './isearch-participants_result';

export class SearchParticipants_Result implements ISearchParticipants_Result {
  
	public Collection?: ISearchParticipants_Item[];
  
	public ErrorMessages: string[] = null;
}


